### DISCUSSION TITLE
Docker - amd64 - All attempts exit with 132

### DISCUSSION THREAD

**Discussion by Normanras** [openobserve/openobserve#1022] [Q&A]
I'm running this on a Proxmox LXC with Docker 24.0 running a few other containers. I'm copying the `docker run` tag in the documentation without changing anything. It pulls the image without an issue but `docker ps -a` shows the container `Exited with code 132`. I've tried a few other tags such as `latest-amd64` and `latest-amd64-simd` but all result in the same thing. 

After looking through some other threads and issues I can confirm that `cat /proc/cpuinfo | grep avx` shows that I have `avx` in my system. Here's the output of `uname -msrv`, if that's helpful:

`Linux 5.15.74-1-pve #1 SMP PVE 5.15.74-1 (Mon, 14 Nov 2022 20:17:15 +0100) x86_64`

I didn't notice any excplict system requirements, so I'm sort of expecting this to work. Will try to Linux binary install, but still want to see if this is issue-worthy for docker installs.

**Comment by descensus** [comment]
Same now on RPI 4 with docker.

For instance, this one works fine: openobserve:v0.8.0, but not v0.8.1
Also tried arm64 images but no difference.

**Comment by hengfeiyang** [comment]
@descensus for building, there is no difference between 0.8.0 and 0.8.1. if `v0.8.0` works, `v0.8.1` also should work, any error logs?

**Reply by descensus** [reply]
Nope, exactly no logs at all when I pull docker logs on the exited container unfortunately.

**Comment by hengfeiyang** [comment]
@Normanras Apologies for the late response. Could you please try using the latest version?

**Reply by Normanras** [reply]
Thank you, @hengfeiyang ! It appears to be working as expected.

**Comment by descensus** [comment]
I extracted the contents of the latest container and try to run the binary manually.

This is on 6.1.0-17-arm64 #1 SMP Debian 6.1.69-1 (2023-12-30) aarch64 GNU/Linux

```
./openobserve
Illegal instruction
```

```
openobserve: ELF 64-bit LSB pie executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, BuildID[sha1]=8fc2c33b68028419d113096e3eb0918a412a3a74, for GNU/Linux 3.7.0, stripped
```

strace ends with:
```
--- SIGILL {si_signo=SIGILL, si_code=ILL_ILLOPC, si_addr=0xaaaad68d1c4c} ---
+++ killed by SIGILL +++
Illegal instruction
```

Is this perhaps a cross compilation issue?

**Reply by descensus** [reply]
Found this https://github.com/openobserve/openobserve/issues/966
